webdriver-spec.html

```diff
@@ -7190,7 +7190,8 @@
         </table>
 
       <p>When required to <dfn>dispatch a keyDown action</dfn> with arguments
-        <var>action object</var> and <var>input state</var> a <a>remote end</a>
+        <var>source id</var>, <var>action object</var>, <var>input state</var>
+        and <var>tick duration</var> a <a>remote end</a>
```

Comment:
If the goal is to synchronise the arguments given to the various dispatch functions, this makes sense. @vkatsikaros?